content/hardware/02.hero/boards/uno-rev3/tutorials/PWM-using-MATLAB-Tutorial/PWM-using-MATLAB-Tutorial.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Controlling PWM output with a potentiometer on the Arduino Uno via MATLAB'
+difficulty: intermediate
+compatible-products: [Arduino Uno Rev 3, Potentiometer]
+description: 'Learn how to dim an LED connceted to the Arduino Uno with MATLAB.'
+tags:
+  - MATLAB
+author: 'Mohammed Hussain Jahangiri'
+hardware:
+  - hardware/boards/arduino-uno-rev3
+software:
+  - MATLAB
+---
+
+## Introduction
+
+In this tutorial, we will use MATLAB to turn on an external LED, and then to adjust the output using MATLAB® Support Package for Arduino® Hardware.
+
+
+### Goals
+
+The goals of this project are:
+
+- Control the Arduino Uno LED with MATLAB.
```

Comment:
```suggestion
- Control the UNO R3 LED with MATLAB.
```

Reply:
After the first time mention of the full name of the product, it is ok to used the product name without the 'Arduino®'